include/myst/pubkey.h

```diff
@@ -6,9 +6,21 @@
 
 #include <stdint.h>
 
+#define MYST_PUBKEY_NAME_SIZE 256
+#define MYST_PUBKEY_MAGIC 0x6f14783e80394815
+
+typedef struct myst_pubkey_header
+{
+    uint64_t magic; /* should be MYST_PUBKEY_MAGIC */
```

Comment:
I think we can get away without a public key header. Just treat public keys as `char* []`.

Reply:
Fixed.